### ISSUE TITLE
Enabling UIA Support in Word Causes Paragraph Indentation Not to be Reported

### ISSUE THREAD

**Issue by msviolafangirl** [nvaccess/nvda#12899]
### Steps to reproduce:
1. Enable UIA support for Word in NVDA avnaced settings.
2. Open a word document.
3. Locate the indented first line of a paragraph.
4. Optionally, press NVDA+F to report formatting at current location.

### Actual behavior:
When I arrow down to the indented first line of a paragraph, NVDA doesn't announce indentation when UIA is enabled. When UIA is disabled, the problem is resolved. Pressing NVDA+F doesn't help.

### Expected behavior:
NVDA should report indentation when I locate the indented first line of a paragraph.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2021.2

#### Windows version:
Windows 10 21H1 x64 build 19043.1266

#### Name and version of other software in use when reproducing the issue:
Office 16/365

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by seanbudd** [comment]
Relates to #12770 - cc @michaelDCurran
